### PR TITLE
Add handler for `unload-feature'

### PR DESCRIPTION
**Pull request by phil-s**

### CODE REVIEW

File: mu4e-thread-folding.el [hunk 1]
```diff
@@ -347,5 +347,18 @@ Unread message are not folded."
 (define-key mu4e-headers-mode-map (kbd "<S-right>") 'mu4e-headers-unfold-all)
 (define-key mu4e-headers-mode-map (kbd "TAB") 'mu4e-headers-toggle-at-point)
 
+(defun mu4e-thread-folding-unload-function ()
```
Comment (rougier) [inline mu4e-thread-folding.el:350]: Why the `function` at the end?

**Comment by rougier** [comment]
Just did some commit, you might need to rebase

File: mu4e-thread-folding.el (hunk 1 above)
Comment (phil-s) [reply mu4e-thread-folding.el:350]: Refer to `C-h f unload-feature`

**Comment by phil-s** [comment]
> Just did some commit, you might need to rebase

You'll need to revert/fix that commit.

https://github.com/rougier/mu4e-thread-folding/commit/0a2688eb43a9b1a604de6d9e440695690bc9330c#commitcomment-47256902

**Comment by rougier** [comment]
Sorry for the mess. I made a new commit to fix the problem.

**Pull request merged**